templates/crate/source.html

```diff
@@ -127,8 +127,15 @@
                 {% else %}
                     {% set file_name = "" %}
                 {% endif %}
-                <div id="source-code" class="pure-u-1 pure-u-sm-17-24 pure-u-md-19-24">
-                    {{- file_content|highlight(file_name)|safe -}}
+                <div id="source-code-container" class="pure-u-1 pure-u-sm-17-24 pure-u-md-19-24">
+                    <div data-nosnippet class="source-code"><pre id="line-numbers"><code>
+                        {%- for line in 1..=file_content.lines().count() -%}
+                            <a href="#{{line|safe}}" id="{{line|safe}}">{{line|safe}}</a>
```

Comment:
Since line is a number, I think `|safe` is useless here.

Reply:
Most likely, this was inspired/taken from rustdoc. I assumed it was used to indicate the string doesn't even need to be changed? 

https://github.com/rust-lang/rust/blob/5c8488605624d67b272953bc21d41db60dbd5654/src/librustdoc/html/templates/source.html#L4

Either way, I can remove it :)

Reply:
Oh I see. Then the `<code>` should be removed but the `~` should be kept.

Reply:
Arf, wrong location for my comment. In this case please remove `|safe`. :)